src/middlewares/user.middleware.js

```diff
@@ -39,7 +39,6 @@ export const checkUserCredentials = async (req, res, next) => {
 };
 export const checkUserIfExist = async (req, res, next) => {
```

Comment:
change this name to the one corresponding to the task that is being done here

Reply:
This was for Hyacente .for me I verified the user ,I did not check